packages/web/src/features/entitlements/server.ts

```
@@ -1,53 +1,107 @@
-const decodeLicenseKeyPayload = (payload: string) => {
+type LicenseKeyPayload = z.infer<typeof eeLicenseKeyPayloadSchema>;
+
+const decodeLicenseKeyPayload = (payload: string): LicenseKeyPayload => {
     const decodedPayload = base64Decode(payload);
     const payloadJson = JSON.parse(decodedPayload);
     return eeLicenseKeyPayloadSchema.parse(payloadJson);
 }
 
+export const getLicenseKey = (): LicenseKeyPayload | null => {
+    const licenseKey = env.SOURCEBOT_EE_LICENSE_KEY;
+    if (licenseKey && licenseKey.startsWith(eeLicenseKeyPrefix)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)